File: 15 more/more.py
import sys
ponudeZaMore={}
for ulaz in sys.stdin:
    imeAgencije,cena=[i for i in ulaz.split()]
    ponudeZaMore.update({int(cena):imeAgencije})  
for ponuda in sorted(ponudeZaMore):
    print(ponudeZaMore[ponuda])   

File: 15 more/more.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace more
{
    class Program
    {
        static void Main(string[] args)
        {
            var ponudeZaMore = new SortedDictionary<int, string>();
            string ulaz;
            while ((ulaz = Console.ReadLine()) != null) {
                string[] elementiUlaza = ulaz.Split();
                ponudeZaMore.Add(int.Parse(elementiUlaza[1]), elementiUlaza[0]);
            }
            foreach (var ponuda in ponudeZaMore) {
                Console.WriteLine(ponuda.Value);
            }
        }
    }
}
